_sass/ecosystem.scss

```diff
@@ -404,3 +404,46 @@
     }
   }
 }
+
+.ecosystem-divider {
+  position: relative;
+  margin-bottom: 4rem;
+  margin-top: 1.5rem;
+  top: 3rem;
+}
+
+.ecosystem, #dropdownSort, #dropdownSortLeft {
+  margin-left: 0 !important;
+}
```

Comment:
You don't have to use `!important` if you put `#dropdownSort` and `#dropdownSortLeft` within an `ecosystem` block. Then those 2 IDs will only apply to the ecosystem page.